@here/harp-mapview/lib/Utils.ts

```diff
@@ -239,9 +239,143 @@ export namespace MapViewUtils {
             groundPlane.constant = 0;
             groundSphere.radius = EarthConstants.EQUATORIAL_RADIUS;
         }
+        if (result !== null) {
+            return result;
+        }
+        return getGazingTargetFromCamera(camera, projection, elevation);
```

Comment:
You should also do this fallback if the tilt is very high (i.e. >89°). Otherwise the raycasted point might be really far away.

Reply:
True, that might be also good idea!